.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+
+# Defines environmental variables
+env:
+  GO_VERSION: '1.22.2'
```

Comment:
To update this myself after speaking with Angela. We don't currently have CLI apps using the auto-Go update action because there is a small problem we need to fix with it first, but this is okay as is for now. It will eventually get updated once we have that action working and in place.